spark/src/test/scala/org/apache/comet/CometCastSuite.scala

```diff
@@ -559,6 +569,12 @@ class CometCastSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     castFallbackTestTimezone(values.toDF("a"), DataTypes.TimestampType, "Unsupported timezone")
   }
 
+  // CAST from BinaryType
+
+  ignore("cast BinaryType to StringType") {
+    // TODO
+  }
```

Comment:
Implementing this new test is unrelated to this PR. I will file a new issue for this.

Reply:
I filed https://github.com/apache/datafusion-comet/issues/377 for binary cast support